packages/core/src/devtools/getResetDebugTree.ts

```diff
@@ -0,0 +1,33 @@
+import { DEBUG_RESET_CLASSES } from '../constants';
+import type { SequenceHash } from '../types';
+import { debugData } from './store';
+import type { DebugSequence } from './types';
+
+export function getResetDebugTree(debugSequenceHash: SequenceHash, parentNode?: DebugSequence) {
+  const resetClass = DEBUG_RESET_CLASSES[debugSequenceHash];
+  if (resetClass === undefined) {
+    return undefined;
+  }
+
+  const debugClassNames = [{ className: debugSequenceHash }];
+
+  const node: DebugSequence = {
+    sequenceHash: debugSequenceHash,
+    direction: 'ltr',
+    children: [],
+    debugClassNames,
+  };
+
+  node.rules = {};
+  node.debugClassNames.forEach(({ className }) => {
+    node.slot = className;
```

Comment:
I am a bit unsure how we could get the `makeResetStyles` variable name here.

Reply:
https://github.com/microsoft/griffel/pull/636/commits/dd2d307fd4ca3cc75b30970957654797cb91d15e